### PR TITLE
[#12][UI] As a user, I can see survey intro - Survey item link

### PR DESCRIPTION
**Pull request by hanam1ni**
## What happened 👀

- Update survey item styling
- Add link to survey detail
 
## Proof Of Work 📹

image

### CODE REVIEW

File: src/components/SurveyList/SurveyItem.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+            data-testid={`survey-item-link-${survey.id}`}
+          >
+            <Image
+              width={20}
+              height={20}
+              src="/icon/chevron-right.svg"
+              alt="right chevron icon"
+            />
+          </div>
+        </Link>
```
Comment (malparty) [inline src/components/SurveyList/SurveyItem.tsx:38]: Should we make the entire survey card clickable? 🤔 
From what I see on most other websites, the whole product card, story, tweet, ... is clickable to ease navigation.

**Review by malparty — APPROVED** [review]
Minor comment about the span of the link. But the rest looks good to me ! :)

File: src/components/SurveyList/SurveyItem.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  return (
+    <>
+      <img
+        src={survey.coverImageUrl}
+        className="w-full h-auto max-h-[360px] mb-8 rounded-lg"
+        alt={`${survey.id} survey image`}
+      />
+      <div className="flex justify-between">
+        <div>
+          <h2 className="text-2xl mb-2">{survey.title}</h2>
```
Comment (hoangmirs) [inline src/components/SurveyList/SurveyItem.tsx:23]: Should this be `h1` in terms of SEO?

File: src/components/SurveyList/SurveyItem.tsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  survey: Survey;
+}
+
+const SurveyItem = ({ survey }: SurveyItemProps) => {
+  return (
+    <>
+      <img
+        src={survey.coverImageUrl}
+        className="w-full h-auto max-h-[360px] mb-8 rounded-lg"
+        alt={`${survey.id} survey image`}
```
Comment (hoangmirs) [inline src/components/SurveyList/SurveyItem.tsx:19]: I think we can use the title here 
```suggestion
        alt={survey.title}
```

File: cypress/integration/Home/index.spec.ts [hunk 4]
```diff
@@ -105,5 +105,29 @@ describe('Home page', () => {
         cy.findByText('Love to hear from you').should('be.visible');
       });
     });
+
+    describe('click on survey item link', () => {
+      it('redirects user to survey detail page', () => {
```
Comment (hoangmirs) [inline cypress/integration/Home/index.spec.ts:110]:
```suggestion
      it('redirects user to the survey detail page', () => {
```

**Review by hoangmirs — APPROVED** [review]
LGTM, just a few minor comments

File: src/components/SurveyList/SurveyItem.tsx (hunk 1 above)
Comment (hanam1ni) [reply src/components/SurveyList/SurveyItem.tsx]: Updated in [34a5526](https://github.com/hanam1ni/react-surveyor/pull/75/commits/34a5526aa6061380d83efb2e799b89da5a745cee)

File: src/components/SurveyList/SurveyItem.tsx (hunk 2 above)
Comment (hanam1ni) [reply src/components/SurveyList/SurveyItem.tsx]: Updated in [34a5526](https://github.com/hanam1ni/react-surveyor/pull/75/commits/34a5526aa6061380d83efb2e799b89da5a745cee)

File: src/components/SurveyList/SurveyItem.tsx (hunk 3 above)
Comment (hanam1ni) [reply src/components/SurveyList/SurveyItem.tsx]: Updated in [34a5526](https://github.com/hanam1ni/react-surveyor/pull/75/commits/34a5526aa6061380d83efb2e799b89da5a745cee)

File: cypress/integration/Home/index.spec.ts (hunk 4 above)
Comment (hanam1ni) [reply cypress/integration/Home/index.spec.ts:110]: Updated in [34a5526](https://github.com/hanam1ni/react-surveyor/pull/75/commits/34a5526aa6061380d83efb2e799b89da5a745cee)

**Pull request merged**